### ISSUE TITLE
Doesn't seem to work on imported modules (ES2015)

### ISSUE THREAD

**Issue by mrmckeb** [tststs/atom-ternjs#204]
In the below example, nothing happens when I type `eventSystem.` - it gives no autocompletion suggestions, even though it's clearly defined as a `new PubSub()`, and that module is imported. It also seems to think that `testController` is an instance of `GameView`. Each exports an ES2015 class.

```
import Controller from './controllers/controller';
import GameModel from './models/gameModel';
import GameView from './views/gameView';
import PubSub from './utils/pubSub';

const eventSystem = new PubSub();

const gameModel = new GameModel(eventSystem);
const gameView = new GameView(eventSystem);
const testController = new Controller(eventSystem, gameModel, gameView);

testController.init();
```

My `.tern-project` file looks like the below, with all appearing when I use the menu option "List analysed files".
```
{
  "ecmaVersion": 6,
  "libs": [
    "browser"
  ],
  "loadEagerly": [
    "src/**/*.js"
  ],
  "plugins": {
    "node": {},
    "complete_strings": {},
    "doc_comment": {
      "fullDocs": true
    },
    "modules": {},
    "es_modules": {}
  }
}
```

Any ideas on what I might be doing wrong?

**Comment by mrmckeb** [comment]
I'm fairly sure this is related to https://github.com/tststs/atom-ternjs/issues/200, as when I went back to v0.12.2, it seems to be working fine.

**Comment by rfreitas** [comment]
I have this same issue. I have tested the same project with the same tern config file on sublime and it works quite well.

**Comment by tststs** [comment]
this should be fixed since 0.14.x.
Feel free to reopen the issue if it's still not working.
